lib/hutch/cli.rb

```diff
@@ -189,6 +189,12 @@ def parse_options(args = ARGV)
           Hutch::Config.pidfile = pidfile
         end
 
+        opts.on('--consumers', 'Show consumers information') do
+          load_app
```

Comment:
Perhaps the description of this option could be "Print consumers information and quit"?